tooling/api/src/tauri.ts

```diff
@@ -9,17 +9,6 @@
  * @module
  */
 
-/** @ignore */
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  interface Window {
-    __TAURI_IPC__: (message: any) => void
-    ipc: {
-      postMessage: (args: string) => void
-    }
-  }
-}
-
```

Comment:
build failed on your own project?

Reply:
trying to build&pack the api package itself.

Reply:
Also, i may be just lost in so many changes here, but using event.listen from it doesn't work for me either, failing with:
```
Uncaught (in promise) invalid args `event` for command `listen`: command listen missing required key event
```
using window.__TAURI_INVOKE__ directly with the same args `listen` injects it works though so i think i am doing something wrong.

Reply:
and lastly, i get stuff like this on Windows:
![grafik](https://github.com/tauri-apps/tauri/assets/30730186/8b886e17-a367-497d-87c5-cfa7043d8dc9)

Reply:
Weird that the build works for me and CI.

I'll check the Windows stuff, seems like the request body isn't being recognized.

Reply:
Should be working now @FabianLars

Reply:
yep, builds fine now and ipc on windows works too. Only the `listen` error still happens. The error on mobile: https://github.com/tauri-apps/tauri/pull/7170#discussion_r1260023780 and on desktop it's still the same one as in the first half of the screenshot weirdly enough.

Reply:
Can you share the code?

Reply:
```ts
listen("deep-link://new-url", console.log);
```
repro steps:
- cd tooling/api
- yarn build
- yarn pack
- copy that tgz to wherever needed
- yarn install in that test project
- add code above, see error in console
- change code to this (copy pasted from event.ts changing invoke to the window api) and it works

```ts
window.__TAURI_INVOKE__<number>("plugin:event|listen", {
    event: "deep-link://new-url",
    windowLabel: "main",
    handler: transformCallback(handler), // handler is just a wrapper around console.log here
  });
```

Reply:
Works on my end anyway. Something we could try is changing the invoke API to use __TAURI_INVOKE__

Reply:
Yeah, works for me in "normal" projects too, just not in npm workspaces because apparently the tgz gets deduplicated&replaced with the released npm package from the workspace.....

Reply:
ok so not our fault :D